.github/workflows/publish.yml

```
@@ -7,8 +7,16 @@ on:
 jobs:
   publish:
-    name: Snapshot build and publish
-    runs-on: ubuntu-latest
+    name: Build and publish KMP library
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        include:
+          - os: ubuntu-latest
+            task: "publishJvmPublicationToMavenCentral publishAndroidReleasePublicationToMavenCentral"
+          - os: macos-latest  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)